.github/workflows/lgtm-approval.yml

```diff
@@ -0,0 +1,34 @@
+name: Approve PR on /lgtm comment
+
+on:
+  issue_comment:
+    types: [created, edited]
+  pull_request_review:
+    types: [submitted, edited]
+  pull_request_review_comment:
+    types: [created, edited]
+
+jobs:
+  approve-pr:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+    if: >
+      (github.event.issue.pull_request && contains(github.event.comment.body, '/lgtm')) ||
```

Comment:
what if i use `/lgtm cancel` or delete my comment? add and remove pr approval should be done in our python code

Reply:
Done in python code, now both options work